bundles/org.openhab.core.model.script/src.moved/test/java/org/openhab/core/model/script/actions/SemanticsTest.java

```diff
@@ -95,8 +104,8 @@ public void setup() throws ItemNotFoundException {
 
     @Test
     public void testGetLocation() {
-        assertThat(Semantics.getLocation(indoorLocationItem), is(indoorLocationItem));
-        assertThat(Semantics.getLocation(bathroomLocationItem), is(bathroomLocationItem));
+        assertNull(Semantics.getLocation(indoorLocationItem));
```

Comment:
For consistency with the constraint-based Assert model:
```suggestion
        assertThat(Semantics.getLocation(indoorLocationItem), is(nullValue()));
```